api/api.js

```diff
@@ -467,6 +467,23 @@ memberRouter.get('/:member_id', async (req, res, next) => {
 	next();
 });
 
+memberRouter.get('/:member_id/update', async (req, res, next) => {
+	if (res.writableEnded) {
+		//do nothing if response already sent
+		next();
+		return;
+	}
+
+	try {
+		await global.setRolesForMember(req.guild, req.member, 'Requested by API');
+		res.send({ status: 'Member update complete' });
```

Comment:
I don't know if it is a good convention, but the convention thus far is to send back the id of the object created or the id of the object acted on for success (resp 200).

Reply:
Also, could we use req.reqMember (if available) in the reason?

Reply:
Fixed ee93a2d.